*** source/javax/activation/MimeTypeParameterList.java.orig	2006-05-01 12:03:04.000000000 -0400
--- source/javax/activation/MimeTypeParameterList.java	2006-05-01 12:04:49.000000000 -0400
***************
*** 197,203 ****
      }
  
      /**
!      * Returns an RFC 2045-complient string representation of this parameter
       * list.
       */
      public synchronized String toString()
--- 197,203 ----
      }
  
      /**
!    * Returns an RFC 2045-compliant string representation of this parameter
       * list.
       */
      public synchronized String toString()
***************
*** 208,218 ****
              String name = (String)i.next();
              String value = (String)parameterValues.get(name.toLowerCase());
  
!             if (buffer.length() > 0)
!             {
!                 buffer.append(';');
!                 buffer.append(' ');
!             }
              buffer.append(name);
              buffer.append('=');
              buffer.append(quote(value));
--- 208,215 ----
              String name = (String)i.next();
              String value = (String)parameterValues.get(name.toLowerCase());
  
!         buffer.append(';');
!         buffer.append(' ');
              buffer.append(name);
              buffer.append('=');
              buffer.append(quote(value));
